jobs/apptypes_test.groovy

```diff
@@ -0,0 +1,61 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+
+name = "apptypes_e2e"
+
+
+  job(name) {
+    description """
+      Watches the e2e-runner repo for a ${config.type} commit
+    """.stripIndent().trim()
+
+    scm {
+      git {
+        remote {
+          github("deis/e2e-runner")
+        }
+        branch('master')
+      }
+    }
+
+
+
+    logRotator {
+      daysToKeep defaults.daysToKeep
+    }
+
+
+
+    triggers {
+      cron('@daily')
+    }
+
+    wrappers {
+      timestamps()
+      colorizeOutput 'xterm'
+    }
+
+    steps {
+      shell 'make docker-test'
+
+      shell """
+        #!/usr/bin/env bash
+        set -eo pipefail
+
+        echo TEST="bps" >> ${defaults.envFile}
+        echo COMPONENT_REPO="e2e-runner" >> ${defaults.envFile}
+        echo UPSTREAM_SLACK_CHANNEL="testing" >> ${defaults.envFile}
+      """.stripIndent().trim()
+
+      downstreamParameterized {
+        trigger(apptype_e2e) {
```

Comment:
this should be the name of the downstream job.  Here we probably want `workflow-test`, right?

Alternately, you could remove the `downstreamParameterized` block and just add another `shell` step calling the default `E2E_RUNNER_JOB` script defined [here](https://github.com/deis/jenkins-jobs/blob/master/common.groovy#L26-L41):

```
shell E2E_RUNNER_JOB
```